.github/workflows/build.yml

```diff
@@ -11,7 +11,6 @@ jobs:
     steps:
     - uses: actions/checkout@v4
     - run: |
-        brew upgrade
```

Comment:
I do not think that you should update this file in this PR. Is that really necessary?